### ISSUE TITLE
Adapt Server to Read new map.json into Memory

### ISSUE THREAD

**Issue by amos1814** [borrascador/rain#293]
Currently the server still reads the map file in json/map.json. There's a bunch of layers (top, middle, bottom, hunting, fishing) stored in memory. These are no longer relevant so I need to go through and remove them, along with references to them in Habitat, Hunting etc. 

The new map format is in map/map.json. Right now this is read into a buffer and sent every time someone logs in. Instead we want it to be read only once.

Estimate: 3 hours
